src/main/drivers/sdio_f4xx.c

```diff
@@ -1654,30 +1654,37 @@ bool SD_GetState(void)
 
 
 /** -----------------------------------------------------------------------------------------------------------------*/
-bool SD_Init(void)
+int SD_Init(void)
 {
     SD_Error_t ErrorState;
 
-    // Initialize SDIO peripheral interface with default configuration for SD card initialization
-    MODIFY_REG(SDIO->CLKCR, CLKCR_CLEAR_MASK, (uint32_t) SDIO_INIT_CLK_DIV);
+    /* Initialize SDIO peripheral interface with default configuration for SD card initialization. */
+    MODIFY_REG(SDIO->CLKCR, CLKCR_CLEAR_MASK, (uint32_t)SDIO_INIT_CLK_DIV);
 
-    if((ErrorState = SD_PowerON()) == SD_OK)                    // Identify card operating voltage
-    {
-        if((ErrorState = SD_InitializeCard()) == SD_OK)         // Initialize the present card and put them in idle state
-        {
-            if((ErrorState = SD_GetCardInfo()) == SD_OK)        // Read CSD/CID MSD registers
-            {
-                // Select the Card - Send CMD7 SDIO_SEL_DESEL_CARD
-                ErrorState = SD_TransmitCommand((SD_CMD_SEL_DESEL_CARD | SD_CMD_RESPONSE_SHORT), SD_CardRCA, 1);
-                MODIFY_REG(SDIO->CLKCR, CLKCR_CLEAR_MASK, (uint32_t) SDIO_CLK_DIV); // Configure SDIO peripheral interface
-            }
-        }
-    }
+    /* Identify card operating voltage. */
+    ErrorState = SD_PowerON();
+    if (ErrorState != SD_OK)
+        return ErrorState;
+
+    /* Initialize the present card and put them in idle state. */
+    ErrorState = SD_InitializeCard();
+    if (ErrorState != SD_OK)
+        return ErrorState;
+
+    /* Read CSD/CID MSD registers. */
+    ErrorState = SD_GetCardInfo();
+    if (ErrorState != SD_OK)
+        return ErrorState;
+
+    /* Select the Card - Send CMD7 SDIO_SEL_DESEL_CARD. */
+    ErrorState = SD_TransmitCommand((SD_CMD_SEL_DESEL_CARD | SD_CMD_RESPONSE_SHORT), SD_CardRCA, 1);
+    /* Configure SDIO peripheral interface. */
+    MODIFY_REG(SDIO->CLKCR, CLKCR_CLEAR_MASK, (uint32_t) SDIO_CLK_DIV);
 
-    // Configure SD Bus width
-    if(ErrorState == SD_OK)
+    /* Configure SD Bus width. */
+    if (ErrorState == SD_OK)
```

Comment:
Betaflight coding style uses 1TBS for braces - apart from function definitions the opening brace is on the same line.